Exercises.Delegate/Exercises.Delegate/Point.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections;
+
+namespace Exercises.Delegate
+{
+    public class Point : IComparable
+    {
+
+        public enum PointRepresentation { Polar, Rectangular }
+
+        private double r, a;
+
+        public Point(double x, double y)
+        {
+            r = RadiusGivenXy(x, y);
+            a = AngleGivenXy(x, y);
+        }
+
+        public Point(double par1, double par2, PointRepresentation pr)
+        {
+            if (pr == PointRepresentation.Polar)
+            {
+                r = par1; a = par2;
+            }
+            else
+            {
+                r = RadiusGivenXy(par1, par2);
+                a = AngleGivenXy(par1, par2);
+            }
+        }
+
+        public double X
+        {
+            get { return XGivenRadiusAngle(r, a); }
+        }
+
+        public double Y
+        {
+            get { return YGivenRadiusAngle(r, a); }
+        }
+
+
+        public double Radius
+        {
+            get { return r; }
+        }
+
+        public double Angle
+        {
+            get { return a; }
+        }
+
+        public void Move(double dx, double dy)
+        {
+            double x, y;
+            x = XGivenRadiusAngle(r, a); y = YGivenRadiusAngle(r, a);
```

Comment:
1. One call per line please
2. Why not using `var` keyword?
3. Follow next pattern:
```
var x = ...;
var y = ...;
```